spec/.github/ISSUE_TEMPLATE/01-compatible-certification-request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Compatible Certification Request
+about: Request certification of a compatibile implementation of MicroProfile Platform release
+title: 'Compatibility certification request for [product/project name] for MciroProfile [plaftorm release version]'
+labels: 'Certification :trophy:'
+assignees: ''
+
+---
+
+- [ ] Organization Name ("Organization") and, if applicable, URL:<br/>
+  // Add here
+- [ ] Product Name, Version and download URL (if applicable):<br/>
+  // Add here
+- [ ] Specification Name, Version and download URL:<br/>
+   // Add here
+- [ ] Public URL of TCK Results Summary for the included MicroProfile component specifications (Config, Fault Tolerance, OpenAPI, Rest Client, Health, Metrics, Open Tracing, JWT):<br/>
+  // Add here
+- [ ] (Optional) Any Additional Jakarta EE Specification TCKs such as the included CDI, JAX-RS, JSON-B, JSON-P, Annotation:<br/>
+  // Add here
+- [ ] Java runtime used to run the implementation:<br/>
```

Comment:
Is it helpful to add the "Java SE 8 or higher" requirement here for clarity?

Reply:
Good idea! added.